assets/js/admin.js

```diff
@@ -57,6 +57,19 @@ jQuery(document).ready(function($) {
         }
     });
 
+    /*
+    Import and Export settings tabs
+     */
+    $('.js-a0-upload-toggle').click(function(){
+        $('#js-a0-upload-file').toggle();
+        $('#js-a0-paste-json').toggle();
+    });
+
+    $('.js-a0-import-export-tabs').click(function (e) {
+        e.preventDefault();
```

Comment:
what's this line doing and why does this function returns an error while the other above doesn't?

Reply:
The HTML is an `<a>` tag so we don't want it to act like a link. 

This function does not return an error ... it doesn't return anything. `e` is for `event`, so stop the default event.